src/OrbitGl/App.cpp

```diff
@@ -1413,6 +1415,34 @@ static std::unique_ptr<CaptureEventProcessor> CreateCaptureEventProcessor(
   return CaptureEventProcessor::CreateCompositeProcessor(std::move(event_processors));
 }
 
+static void FindAndAddFunctionToStopUnwindingAt(
+    const std::basic_string<char>& function_name, const std::basic_string<char>& module_name,
```

Comment:
`std::basic_string<char>`? You mean `std::string`?

Reply:
Yeah. That was Clion...